### DISCUSSION TITLE
Proxmox Helper scripts error Internet not connected

### DISCUSSION THREAD

**Discussion by flnf** [community-scripts/ProxmoxVE#178] [Q&A]
I have a clean installation of proxmox 8.2.
Search domain=flnf.lan (internal network), DNS server 1=192.168.1.1 (my router), DNS Server 2=1.1.1.1
When I try to install any of the scripts, for example the pihole one, I always get Internet not connected during installation. I install in advanced mode with a fixed IP (192.168.1.250) gateway=192.168.1.1 hostname=pihole, search dns=flnf.lan and dns=192.168.1.1
Can you help me? Could it be because I don't have a DNS, for example bind9, installed on the proxmox server, or on a virtual machine, or on another computer outside of proxmox to resolve the local domain flnf.lan?

**Comment by MickLesk** [comment]
pihole need dns: 
PIHOLE_DNS_1=8.8.8.8
PIHOLE_DNS_2=8.8.4.4
